terraform/modules/hub/files/prometheus/prometheus.yml

```diff
@@ -65,6 +65,13 @@ scrape_configs:
       - source_labels: [__meta_ec2_tag_Cluster]
         replacement: $${1}_journalbeat
         target_label: job
+  - job_name: frontend
+    scheme: 'https'
+    tls_config:
+      insecure_skip_verify: true
+    dns_sd_configs:
+      - names:
+          - 'frontend.hub.local'
```

Comment:
I don't like this name but I can't think of a better one

We use the external URLs everywhere else so it feels like we should use that?

I'll add some suggestions but if you have concerns I won't let this block the change

Reply:
This is, and should be, a private DNS zone, so I don't think it should use `${root_domain}`

Reply:
Yes, however everywhere else we have private DNS zones which _do_ use the root domain, this is how we get trusted TLS using the public certs.

Because we don't care about TLS in this case then not using the root domain is fine. However then I would suggest using `.vpc` instead of `.local` unless there is another reason why we must use `.local`.

I will raise a suggestion for `.vpc`.

Reply:
can you explain why you think `.vpc` is better?

Reply:
When I think of local I think of loopback/localhost, this might not be a good enough reason in itself, your choice.

A counterargument to `.vpc` instead of `.local` is that kops etc use `.local` and that it is reserved by ICANN

Reply:
yeah i don't think this is enough of a reason for me. I'm going to leave it as is. can I get a ✅ please? 😄